setup.py

```diff
@@ -244,6 +244,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 ]
 databricks = [
     'requests>=2.26.0, <3',
+    'azure-identity>=1.3.1',
```

Comment:
I removed it, primarily because of the explicit support for managed identity. Primary reason for its use was for caching of tokens, etc. But it was easy to implement

Reply:
Good idea not to add yet another dependeny :)